airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.1
```

Comment:
Oh you don't need this, [this job](https://github.com/airbytehq/airbyte/actions/workflows/publish-cdk-command-manually.yml) will do it for you.

I did the same and had to amend it, please check:
https://github.com/airbytehq/airbyte/pull/38237/files